README.md

```diff
@@ -33,6 +33,7 @@ Download and install the latest release of Go, at least Go 1.11:
 https://golang.org/
 
 You will also need Git. (`brew install git`, `apt-get install git`, etc).
+If you're building on Windows, you also need to install and have gcc setup correctly (easiest way via mingw64).
```

Comment:
Do you think a link to mingw64 would be helpful (not a windows person)

Reply:
https://gcc.gnu.org/install/binaries.html
Maybe thats better, youre right 👍

Reply:
At the moment we don't need libs so TDM-GCC64 works fine as well. Easier to setup and people don't have to start using new shell.

Reply:
@egonelbre TDM-GCC64 is using mingw under the hood afaik

Reply:
Sure, either is fine.